content/rosa/ip-addressing-and-subnets/index.md

```diff
@@ -0,0 +1,101 @@
+---
+date: "2023-01-06"
+title: "VPC and Subnet IP Address Considerations with ROSA"
+tags: ["rosa","vpc", "subnet"]
+authors:
+   - Thatcher Hubbard
+---
+
+# VPC and Subnet IP Address Considerations with ROSA
+
+ROSA clusters can be built to be highly available using the fundamental capability that underlies most HA configurations on AWS: Availability Zones. By spreading the resources of a cluster across three separate (but regionally co-located) datacenters, ROSA users can ensure the cluster continues to run even if an entire AWS AZ goes down.
+
+This capability comes with a few challenges and considerations around IP addressing that this article will attempt to explain and provide options and best practices around.
+
+## ROSA and network CIDRs
+
+Configuring a ROSA cluster requires the provision of three non-overlapping CIDR ranges, as is typical for any Kubernetes installation:
+
+1. The Machine CIDR range (the range that node IP addresses are assigned from)
+1. The Service CIDR range (the range that OpenShift will assign Service IPs from)
+1. The Pod CIDR range (the range that OpenShift will assign Pod IPs from)
+
+The Service and Pod CIDR ranges are typically safe to overlap with other clusters because they are used exclusively for intra-cluster communication.
```

Comment:
"However they should never conflict with other non-OCP resources on your network, while they're non-routable, if they overlap with another service elsewhere on the network, they won't be able to route to that service"